framework/src/modules/chain/submodules/loader.js

```diff
@@ -915,7 +915,7 @@ __private.sync = function(cb) {
 			__private.blocksToSync = 0;
 
 			library.logger.info('Finished sync');
-			if (!components.cache.cacheReady) {
+			if (components.cache.options.enabled && !components.cache.cacheReady) {
```

Comment:
Same as above

```suggestion
			if (components.cache.options.enabled) {
```

Reply:
Cache can be enabled and not ready (for example redis was not up at boot or went down)